yt/frontends/amrex/tests/test_outputs.py

```diff
@@ -445,6 +446,96 @@ def test_maestro_parameters():
     assert type(ds.parameters["s0_interp_type"]) is int  # noqa: E721
 
 
+quokka = "quokka_RadiatingParticles_plt026"
```

Comment:
Given how you have https://github.com/Rongjun-ANU/README-of-yt-frontend-for-QUOKKA/tree/main set up, it will be easier to just use paths here: 

```suggestion
quokka = "quokka/RadiatingParticles/plt026"
```

Then the `sample` folder from https://github.com/Rongjun-ANU/README-of-yt-frontend-for-QUOKKA/tree/main can be copied to a `quokka` directory within yt's test data directory (rather than renaming, e.g., `sample/RadiatingParticles/plt026` to `sample_RadiatingParticles_plt026`).